app/js/lib/foxbox/foxbox.js

```diff
@@ -180,32 +149,29 @@ export default class Foxbox extends Service {
         // we have no record of previous registrations, we schedule
         // a retry.
         if (!this[p.boxes].length &&
-            !this[p.settings]._localOrigin &&
```

Comment:
Hmm, I thought we had public ```localOrigin/hostName``` and ```tunnelOrigin/hostName``` properties for some time already, but nevermind :)